hvantk/core/converters.py

```
@@ -0,0 +1,157 @@
+
+    Returns
+    -------
+    ad.AnnData
+        AnnData object with X as float32.
+    """
+    # Lazy Hail import (via hail_context to apply numpy compat shim first)
+    from hvantk.core.hail_context import hl  # noqa: F811
+
+    # Resolve keys
+    if row_key is None:
+        row_key = list(mt.row_key)[0]
```

Ruff (0.15.9)

[warning] Prefer `next(iter(mt.row_key))` over single element slice

Replace with `next(iter(mt.row_key))`

(RUF015)